Add support Office PowerPoint 2007 (macro-enabled) slide show

Borewit opened the pull request:
Normalized MIME-types to lowercase.
MIME-type are defined to be case insensitive.

Add support for:
- `.ppsx`, `application/vnd.openxmlformats-officedocument.presentationml.slideshow`
- `.ppsx`, `application/vnd.ms-powerpoint.slideshow.macroEnabled.12`

MIME type reference: [Register the 2007 Office system file format MIME types on servers](https://learn.microsoft.com/en-us/previous-versions/office/office-2007-resource-kit/ee309278%28v=office.12%29)

Resolves sindresorhus/file-type#746

PRR24 commented:
Not related to the task, but reviewing the changes I noticed that all macroEnabled.12 mime-s are in camelCase except
```application/vnd.ms-excel.template.macroenabled.12```

package.json also got unrelated "packageManager" entry. Not sure, if it is intentional or a mistake.

Other than that, looks exactly as expected, nice job.

Borewit commented:
Thx! @PRR24  

- Removed `packageManager`, was not intentional .
- MIME are defined to be case insensitive. Adjusted other types be lower case, and the switch case insensitive.
